OData/src/System.Web.OData/OData/ODataUriFunctions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.OData.Core.UriParser;
```

Comment:
I have considered that, but as you can see, these methods are not extension methods, they are just static.
But I don't rellay mind if you prefer to put them in HttpConfigurationExtensions.